Index: internal/envtest/webhooks.go
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/internal/envtest/webhooks.go b/internal/envtest/webhooks.go
--- a/internal/envtest/webhooks.go	(revision Staged)
+++ b/internal/envtest/webhooks.go	(date 1627590181567)
@@ -75,10 +75,11 @@
 		klog.Fatalf("Failed to append controlplane controller webhook config: %v", err)
 	}
 	return envtest.WebhookInstallOptions{
-		MaxTime:            20 * time.Second,
-		PollInterval:       time.Second,
-		ValidatingWebhooks: validatingWebhooks,
-		MutatingWebhooks:   mutatingWebhooks,
+		MaxTime:                      20 * time.Second,
+		PollInterval:                 time.Second,
+		ValidatingWebhooks:           validatingWebhooks,
+		MutatingWebhooks:             mutatingWebhooks,
+		LocalServingHostExternalName: "docker.for.mac.localhost",
 	}
 }
 
